framework/src/modules/chain/loader.js

```diff
@@ -100,6 +100,7 @@ class Loader {
 		// TODO: Add target module to procedure name. E.g. chain:getTransactions
 		const { data: result } = await this.channel.invoke('network:request', {
 			procedure: 'getTransactions',
+			data: {},
```

Comment:
This feels a bit hacky, could we create an issue to deal with this on network module side and reference it here?

Reply:
This is actually not related to the network module
maybe, we should fix it on the transport side